crypter.go

```diff
@@ -403,9 +423,20 @@ func (ctx *genericEncrypter) Options() EncrypterOptions {
 	}
 }
 
-// Decrypt and validate the object and return the plaintext. Note that this
-// function does not support multi-recipient, if you desire multi-recipient
+// Decrypt and validate the object and return the plaintext. This
+// function does not support multi-recipient. If you desire multi-recipient
 // decryption use DecryptMulti instead.
+//
+// The decryptionKey argument most have one of these types:
+//   - *rsa.PrivateKey
+//   - *ecdsa.PrivateKey
+//   - []byte (will be interpreted as the bytes of a symmetric key)
+//   - string (will be interpreted as the bytes of a symmetric key)
+//   - *JSONWebKey
+//   - JSONWebKey
+//   - *JSONWebKeySet
+//   - JSONWebKeySet
```

Comment:
Here you list the pointer types first; on line 126/127 you listed the pointer type second.